src/JsonSchema/Generator/Normalizer/NormalizerGenerator.php

```diff
@@ -195,4 +198,11 @@ protected function normalizeMethodStatements(Expr\Variable $dataVariable, ClassG
             new Stmt\Expression(new Expr\Assign($dataVariable, new Expr\New_(new Name('\\stdClass')))),
         ];
     }
+
+    protected function isReadOnlyProperty(Property $property): bool
```

Comment:
I will prefer to have a property inside `Property` class to store this value and having this logic inside a Guesser

Reply:
Very good point, totally missed that. We have that field already on `Property` ;-)